src/messages.js

```diff
@@ -74,23 +74,29 @@ module.exports = Tag => ({
     ],
     "lookup_challengers select_one": [
         Tag `sólo uno de los ${'num'}, tendrás que elegir a quién`,
-        Tag `deberás elegir sólo uno de los ${'num'}`
+        Tag `deberás elegir sólo uno de los ${'num'}`,
+        Tag `sólo a uno le puedes partir la madre, elige uno de los ${'num'}`
     ],
 
 
     // ########## Reported Results ################
     "reported_result valid": [
         Tag `Enterado... <@${'winner'}> le ganó a <@${'loser'}> ${'highScore'} - ${'lowScore'}.`,
         Tag `Órale, así que <@${'loser'}> perdió ${'highScore'} - ${'lowScore'} contra <@${'winner'}>... ¿quién lo habría imaginado?.`,
-        Tag `¿${'highScore'} - ${'lowScore'} contra <@${'loser'}>?. Ése <@${'winner'}> va subiendo como la espuma.`
+        Tag `¿${'highScore'} - ${'lowScore'} contra <@${'loser'}>?. Ése <@${'winner'}> va subiendo como la espuma.`,
+        Tag `:musical_note: Nada... ese <@${'loser'}> no trae nada... :musical_note: ¡Buen trabajo, <@${'winner'}>!`,
```

Comment:
All `reported_result valid` messages must include the `highScore `, `lowScore `, `loser ` & `winner` becasue it works as a confirmation for the user, so in case the user see something wrong in the interpretation of the bot he can edit the message or rise an issue.

Reply:
Got it, I'll update it.